src/Common/RJSF/templates/ObjectFieldTemplate.tsx

```diff
@@ -54,8 +57,19 @@ const Field = ({
                 return true
             }
             try {
-                const value = JSONPath({ path: hiddenSchemaProp.match, json: formData })?.[0]
-                const isHidden = value === undefined || hiddenSchemaProp.condition === value
+                const hiddenSchema = parseSchemaHiddenType(hiddenSchemaProp)
+                if (!hiddenSchema.path) {
```

Comment:
Suggestion: Can move this validation into validators.tsx if it can be re-used

Reply:
seems a little too redundant atm